docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3-slim
+LABEL maintainer="OpenManageAnsible@Dell.com"
```

Comment:
Open Manage does holds maintaining rights of the Image customer Creates, remove this option LABEL

Reply:
@rajeevarakkal I am OK to remove this, however in future when we are publishing pre-built image to Dockerhub we may need to add it back. Also i dont see any problem in having this label as it is just a metadata to an image.

Reply:
@saishivarcr , Some point when we have the docker image posted, we may need to bring this back and it make sense that point of time. Let get it done that point of time.  Currently customer may opt to customize the script and chances are there customer will be adding multiple packages in to this script and the generate the docker build out of it. IMHO it is risky to take maintainer rights of any such images. If you still feels this needs to be kept, i would request you to get a sign off from Security Team. I am good in that case

Reply:
@rajeevarakkal I will remove this.